cub/device/device_radix_sort.cuh

```diff
@@ -66,7 +66,7 @@ namespace cub {
  * half-precision floating-point type.  Although the direct radix sorting
  * method can only be applied to unsigned integral types, DeviceRadixSort
  * is able to sort signed and floating-point types via simple bit-wise transformations
- * that ensure lexicographic key ordering. For floating-point types -0.0 and +0.0 are
+ * that ensure lexicographic key ordering. DeviceRadixSort is stable For floating-point types -0.0 and +0.0 are
```

Comment:
This needs to be blended into the text a bit more. It reads as "Device radix sort is stable for floating-point types", which is true, but the results are stable for non-floating-point types, too,

Can we switch this to "DeviceRadixSort is stable, and for floating-point types..." (or just make them separate sentences)?

Reply:
sure